acceptance/allocator_terraform/load.tf

```diff
@@ -4,6 +4,10 @@ variable "example_block_writer_instances" {
   default = 1
 }
 
+variable "load_duration" {
+  default = "0" # unlimited
```

Comment:
Why do you need this? Appears to me that the duration is really governed by the test runner (assuming you can be off by a few minutes). You could make the tests omit the duration completely and I don't see what would be lost.